plugin/src/automation/lift/lift.ml

```diff
@@ -341,7 +347,7 @@ let repack c env lifted typ sigma =
      let args = unfold_args typ in
      let sigma, typ_red = specialize_delta_f env f args sigma in
      sigma, mkLetIn (Anonymous, lifted, typ, eta_prod_rec (mkRel 1) (shift typ_red))
-  | SwapConstruct _ ->
+  | SwapConstruct _ | UnpackSigma ->
```

Comment:
Think about and discuss this. Why don't we need this, if we don't need this? Do we perhaps need this in the backwards direction? Is this handled by our constructors?